config/betterquesting/resources/supersymmetry/lang/en_us.lang

```diff
@@ -77,55 +77,55 @@ susy.quest.db.9.desc=Using our §6Crafting Table§r you can now construct §7Fli
 susy.quest.db.12.title=Stone Tools
-susy.quest.db.12.desc=Now, you should be able to craft your first §7tools§r in your own crafting inventory! A pickaxe and an axe should suffice for now, although a shovel is nice for getting through dirt.%n%nNote: To find the recipe for stone tools, you should check the recipe for their iron variant (or simply by pressing §6[R]§r on the items on the right-hand side) and turn to the §6last§r page of the JEI.
+susy.quest.db.12.desc=Now, you should be able to craft your first §7tools§r in your crafting inventory! A pickaxe and an axe should suffice for now, although a shovel is nice for getting through dirt.%n%nNote: To find the recipe for stone tools, you should check the recipe for their iron variant (or simply by pressing §6[R]§r on the items on the right-hand side) and turn to the §6last§r page of the JEI.
 susy.quest.db.13.title=Copper
-susy.quest.db.13.desc=Copper can be found pretty much everywhere in the Earth's crust, and even in the deeper layers. Sometimes you might even find some §6Native Copper Deposits§r, which contains coppers in its elemental form.%n%nYou can get coppers from §eChalcopyrite§r, §8Chalcocite§r, §2Malachite§r, §6Bornite§r, or §4Tetrahedrite§r veins. However, the Primitive Smelter can only process ores with a single type of metal inside, which means that §c§lONLY§r Chalcocite and Malachite can be processed by this.%n%nFinding a §6Tiny Pile of Copper Dust §ron the ground means that there is a deposit under it. If you are having trouble finding a specific deposit, look it up in the JEI database, you will find a page showing where it is found. %n%nLet's start by getting 30 copper ingots.%n%n§c§lAs a very important rule of thumb, quest retrieval numbers are arbitrary. You can follow the suggestion here, but trust me when I say you will need a whole lot more Copper.
+susy.quest.db.13.desc=Copper can be found pretty much everywhere in the Earth's crust, and even in the deeper layers. Sometimes you might even find some §6Native Copper Deposits§r, which contain copper in its elemental form.%n%nYou can get coppers from §eChalcopyrite§r, §8Chalcocite§r, §2Malachite§r, §6Bornite§r, or §4Tetrahedrite§r veins. However, the Primitive Smelter can only process ores with a single type of metal inside, which means that §c§lONLY§r Chalcocite and Malachite can be processed by this.%n%nFinding a §6Tiny Pile of Copper Dust §ron the ground means that there is a deposit under it. If you are having trouble finding a specific deposit, look it up in the JEI database, and you will find a page showing where it is found. %n%nLet's start by getting 30 copper ingots.%n%n§c§lAs a very important rule of thumb, quest retrieval numbers are arbitrary. You can follow the suggestion here, but trust me when I say you will need a whole lot more Copper.
```

Comment:
contain*s* copper

Reply:
The subject here is "Native Copper Deposits" which is plural, so we need the plural verb form "contain" and not "contains".